aomi/seed.py

```diff
@@ -29,22 +30,35 @@ def ensure_mounted(client, backend, mount):
                          (mount, m.group('path')))
 
 
+def ensure_auth(client, auth):
+    """Will ensure a particular auth endpoint is mounted"""
+    backends = client.list_auth_backends()['data'].keys()
+    backends = [x.rstrip('/') for x in backends]
+    if auth not in backends:
+        client.enable_auth_backend(auth)
+
+
 def var_file(client, secret, opt):
     """Seed a var_file into Vault"""
     path = "%s/%s" % (secret['mount'], secret['path'])
     var_file_name = hard_path(secret['var_file'], opt.secrets)
+    secret_file(var_file_name)
     varz = yaml.load(open(var_file_name).read())
     if 'var_file' not in secret \
        or 'mount' not in secret \
        or 'path' not in secret:
         problems("Invalid generic secret definition %s" % secret)
```

Comment:
Maybe say what is missing in the problems. Maybe break this validate if statement in to a function that returns an error message if there is any problems.